src/i18n/translations/kr/translation.json

```diff
@@ -1,64 +1,73 @@
 {
-  "404": {
-    "statusCode": "404",
-    "errorMessage": "잘못된 URL이거나, 존재하지 않는 페이지입니다. <2>홈페이지로 돌아가기</2>"
+  "404" : {
+    "statusCode" : "404",
+    "errorMessage" : "잘못된 URL이거나, 존재하지 않는 페이지입니다. &lt;2&gt;홈페이지로 돌아가기&lt;/2&gt;"
```

Comment:
I think this can just be made `<2>` since its in a string, but i'm really not sure what this is...

Reply:
i totally spaced this bug being a possibility here. pushed up a fix